appengine/flexible/multiple_services/api_gateway/README.md

```diff
@@ -0,0 +1,56 @@
+# Python Google Cloud Microservices Example - API Gateway
+
+This example demonstrates how to deploy multiple python services to [App Engine flexible environment](https://cloud.google.com/appengine/docs/flexible/)
+
+## To Run Locally
+
+1. You will need to install Python 3 on your local machine
+
+2. Install virtualenv
+```Bash
+$ pip install virtualenv
+```
+
+3. To setup the environment in each server's directory:
+```Bash
+$ virtualenv -p python3 env
+$ source env/bin/activate
+$ pip install -r requirements.txt
+$ deactivate
+```
+
+4. Change the url's in the API gateway `gateway/api_gateway.py` to the
+development ports specified in line `gateway/api_gateway.py:6`
+```Python
+development ports {static: 8003, flask1: 8001, flask2: 8002}
+```
+
+## To Deploy to App Engine
+
+### YAML Files
+
+Each directory contains an `app.yaml file`.  This file will be the same as if
+each service were its own project except for the `service` line. For the gateway:
```

Comment:
I would prefer to remove the line "This file will be the same as if
 +each service were its own project except for the `service` line.", perhaps "These files all describe a separate App Engine service within the same project".